CMakeLists.txt

```diff
@@ -60,14 +60,16 @@ set(CMAKE_CXX_FLAGS_FASTDEBUG "-O3 -g")
 set(CMAKE_CXX_FLAGS_DEBUG     "-O0 -g")
 
 # Include directories
-include_directories (core mss sim)
+include_directories (core mss sim mavis/mavis fusion/fusion)
```

Comment:
The reason I had `mavis` as a requirement for the include directive (`#include "mavis/Mavis.h"`) was to prevent header file naming collisions if they happened.  I don't have complete control on Mavis, so if someone added `mavis/CommonTypes.hpp` and we had `CommonTypes.hpp` the compiler wouldn't get confused.   Can this rule be put back?

Reply:
Yes, of course.

I'll also make sure the fusion headers are consistent with this in olympia. 
e.g. #include "fusion/FusionWhatsIt.hpp"

It will look like this now:

# Include directories
include_directories (core mss sim)
include_directories (SYSTEM fusion)
include_directories (SYSTEM mavis)
include_directories (SYSTEM stf_lib)